lib/contentful_delivery/delivery.ex

```diff
@@ -0,0 +1,269 @@
+defmodule Contentful.Delivery do
```

Comment:
I believe this file is a bit loaded with functionality. I can see an API client, configuration, API error handling, and some request utilities. Any plan to break this up into smaller chunks? Would help me to reason better about the different responsibilities.

Reply:
Most of the functions in there will be moved out as soon as I get `Contetnful.Management` going.